### PR TITLE
DOCS-2889: Turn access arm tutorial into how-to

### PR DESCRIPTION
**Pull request by sguequierre**
* adds new gifs 
* makes shorter
https://docs-test.viam.dev/3650/how-tos/move-robot-arm/

### CODE REVIEW

File: docs/how-tos/move-robot-arm.md
Comment (sguequierre) [inline docs/how-tos/move-robot-arm.md:71]:
```suggestion
```

File: docs/how-tos/move-robot-arm.md
Comment (sguequierre) [inline docs/how-tos/move-robot-arm.md:70]:
```suggestion
```

File: docs/how-tos/move-robot-arm.md
Comment (sguequierre) [inline docs/how-tos/move-robot-arm.md:80]:
```suggestion
# Joint Positions of arm1
```

File: docs/how-tos/move-robot-arm.md
Comment (sguequierre) [inline docs/how-tos/move-robot-arm.md:89]:
```suggestion
// Joint Positions of arm1
```

File: docs/how-tos/move-robot-arm.md
Comment (npentrel) [inline docs/how-tos/move-robot-arm.md:155]:
```suggestion
{{% tablestep link="/appendix/apis/components/arm/#movetoposition" %}}
```

**Review by npentrel — CHANGES_REQUESTED** [review]

**Comment by npentrel** [comment]
Love the GIFs and how this is looking. I think @JessamyT might actually be a better reviewer so tagging her here as well

File: docs/how-tos/move-robot-arm.md
Comment (JessamyT) [reply docs/how-tos/move-robot-arm.md:110]: I'm going to ask Esha and Ray whether we should just take out the pose command stuff from this tutorial anyway since they are deprecated and the motion service does the same thing.

File: docs/how-tos/move-robot-arm.md
Comment (JessamyT) [reply docs/how-tos/move-robot-arm.md:151]: Agree, esp. since 30 degrees actually isn't so small :D

**Comment by JessamyT** [comment]
Big fan of the GIFs!

**Comment by JessamyT** [comment]
I think it'd be good to show the full code at the end.
Also [nit]  consider, in the code snippets, either removing the line numbers or adding like `data-start="94"` so they better reflect what the user will do.

File: docs/how-tos/move-robot-arm.md
Comment (sguequierre) [reply docs/how-tos/move-robot-arm.md:63]: Ok

File: assets/how-tos/access-arm/config.png
Comment (sguequierre) [reply assets/how-tos/access-arm/config.png:1]: Deleting the image anyways

File: docs/how-tos/move-robot-arm.md
Comment (sguequierre) [reply docs/how-tos/move-robot-arm.md:132]: Thanks!

File: docs/how-tos/move-robot-arm.md
Comment (sguequierre) [reply docs/how-tos/move-robot-arm.md:151]: haha yeah just copying from old tutorial

File: docs/how-tos/move-robot-arm.md
Comment (sguequierre) [reply docs/how-tos/move-robot-arm.md:38]: Are you sure it's wise to add in the info about a fake arm? I had it in there but decided to remove it because you can't see any of the movement commands, it seems a bit pointless.

File: docs/how-tos/move-robot-arm.md
Comment (sguequierre) [reply docs/how-tos/move-robot-arm.md:115]: @JessamyT can you check what I wrote on line 114? not entirely sure if that's the best description

File: docs/how-tos/move-robot-arm.md
Comment (npentrel) [inline docs/how-tos/move-robot-arm.md:38]: you also need a machine to run viam-server on...

File: docs/how-tos/move-robot-arm.md
Comment (npentrel) [inline docs/how-tos/move-robot-arm.md:42]:
```suggestion
- A single-board computer or other computer to run `viam-server`.
- A Linux, macOS or Windows computer that can run SDK code.
```

File: docs/how-tos/move-robot-arm.md
Comment (npentrel) [inline docs/how-tos/move-robot-arm.md:24]: I think this actually looks weird here I think let's remove
```suggestion
```

File: docs/how-tos/move-robot-arm.md
Comment (npentrel) [inline docs/how-tos/move-robot-arm.md:32]: To avoid having two alerts here, let's put this with the Move the arm section

File: docs/how-tos/move-robot-arm.md
Comment (npentrel) [inline docs/how-tos/move-robot-arm.md:7]: can you add your gif as a preview here?

**Comment by sguequierre** [comment]
> I think it'd be good to show the full code at the end. Also [nit] consider, in the code snippets, either removing the line numbers or adding like `data-start="94"` so they better reflect what the user will do.

removed line numbers

**Review by npentrel — APPROVED** [review]
LGTM once Jessamy approves

**Comment by JessamyT** [comment]
Thoughts on showing the full script?

Some sections still have line numbers so the mix looks odd. Either remove from all or add to all with somewhat correct-ish numbers.

**Comment by sguequierre** [comment]
> Thoughts on showing the full script?
> 
> Some sections still have line numbers so the mix looks odd. Either remove from all or add to all with somewhat correct-ish numbers.

I don't think the full script is necessary for a how-to. unless you feel strongly otherwise I'm fine with it as-is
Removed remaining line numbers

**Review by JessamyT — APPROVED** [review]
LGTM

**Pull request merged**
